corehq/apps/api/accounting.py

```diff
@@ -181,7 +181,7 @@ class InvoiceResource(ModelResource):
 
     class Meta(AccountingResourceMeta):
         queryset = Invoice.api_objects.all().order_by('pk')
-        fields = ['id', 'tax_rate', 'balance', 'date_due', 'date_paid', 'date_created', 'date_received',
+        fields = ['id', 'tax_rate', 'balance', 'date_due', 'date_paid', 'date_created',
```

Comment:
@lwyszomi I'm going to be removing the ```date_received``` field from ```Invoice```.  Will this break any of the Salesforce code that uses this API?